frontend/src/review/review-doc/review-doc.component.ts

```diff
@@ -0,0 +1,170 @@
+import { Component, OnInit, ViewChild, ElementRef, OnDestroy } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+import { MarkedTextModel } from '../models/marked-text.model';
+import { TooltipComponent } from '../tooltip/tooltip.component';
+
+import { Store } from '@ngrx/store';
+import { Subscription } from 'rxjs';
+
+import * as Actions from '../store/review.actions';
+import * as Selectors from '../store/review.selectors';
+import { ReviewService } from '../services/review.service';
+import { CommentModel } from 'src/app/models/comment.model';
+import { DocumentService } from 'src/document/services/document.service';
+import { flatMap } from 'rxjs/operators';
+
+
+@Component({
+    selector: 'app-review-doc',
+    templateUrl: './review-doc.component.html',
+    styleUrls: ['./review-doc.component.scss']
+})
+export class ReviewDocComponent implements OnInit, OnDestroy {
+  @ViewChild(TooltipComponent, { read: ElementRef }) tooltipComponent: ElementRef;
+
+  id: string;
+  pageNr: number;
+
+  subscription: Subscription;
+
+  tooltipPositionStyles: object;
+  markedText: MarkedTextModel[];
+  mouseEvent: MouseEvent;
+
+  comments: CommentModel[];
+
+  constructor(
+    private route: ActivatedRoute,
+    private store: Store<any>,
+    private reviewService: ReviewService,
+    private docService: DocumentService,
+    ) {
+    this.id = this.route.snapshot.paramMap.get('id');
```

Comment:
=+ (konwersja na number)

Reply:
id tutaj jest z mongodb i to id nie jest intem